pkg/controller/appbindings.go

```diff
@@ -58,6 +58,8 @@ func (c *StashController) applyBackupAnnotationLogicForAppBinding(ab *appCatalog
 		return fmt.Errorf("failed to create reference of %s %s/%s. Reason: %v", ab.Kind, ab.Namespace, ab.Name, err)
 	}
 
+	targetAppGroup, targetAppResource := ab.GroupResource()
```

Comment:
These are optional & it should be ok for these to be empty. Just not usable in that case.